src/dropdown.ts

```diff
@@ -318,15 +318,15 @@ export default class Dropdown extends LitElement implements FormControl {
     }
 
     for (const option of this.#optionElements) {
-      this.#isProgrammaticSelection = true;
+      this.#isSelectionFromValueSetter = true;
```

Comment:
This field is only set in one place now. That let me give it a more descriptive name.